cypress/support/config/settings.js

```diff
@@ -1226,16 +1226,16 @@ module.exports = () => ({
       featureIndexPage: {
         environments: {
           live: {
-            paths: [],
+            paths: ['/bengali/53988070'],
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/bengali/sport-2306692'],
```

Comment:
404 error page

Reply:
Well spotted, it was missing a number at the end. Fixed now.